src/page-modules/assistant/details/index.tsx

```diff
@@ -35,6 +37,12 @@ export function AssistantDetails({ tripPattern }: AssistantDetailsProps) {
     tripSearchParams.append('filter', router.query.filter as string);
   }
 
+  const requireTicketBooking = tripPattern.legs.some(
```

Comment:
Is the same done in the app to know if booking is required?

Reply:
In the app, there is a function to check whether a leg in the trip requires a ticket that can not be must be bought through the app. https://github.com/AtB-AS/mittatb-app/blob/eb680679c754e8e56848ffe7983779c0794468ba/src/travel-details-screens/components/Trip.tsx#L106-L109.

The solution I used in this PR is pretty much the same approach to determine whether the BookingSection should appear. https://github.com/AtB-AS/planner-web/blob/a4453ced8626eba2d075255fe06146e06e1424a8/src/page-modules/assistant/details/trip-section/index.tsx#L75-L79